src/mxarray.jl

```diff
@@ -686,11 +686,10 @@ end
 
 # deep conversion from MATLAB variable to Julia array
 
-jvalue(mx::MxArray, ty::Type{Array})  = jarray(mx)
-jvalue(mx::MxArray, ty::Type{Vector}) = jvector(mx)
-jvalue(mx::MxArray, ty::Type{Matrix}) = jmatrix(mx)
-jvalue(mx::MxArray, ty::Type{Number}) = jscalar(mx)::Number
-jvalue(mx::MxArray, ty::Type{String}) = jstring(mx)::String
-jvalue(mx::MxArray, ty::Type{Dict}) = jdict(mx)
-jvalue(mx::MxArray, ty::Type{SparseMatrixCSC}) = jsparse(mx)
-
+convert(::Type{Array}, mx::MxArray)  = jarray(mx)
+convert(::Type{Vector}, mx::MxArray) = jvector(mx)
+convert(::Type{Matrix}, mx::MxArray) = jmatrix(mx)
+convert(::Type{Number}, mx::MxArray) = jscalar(mx)
+convert(::Type{String}, mx::MxArray) = jstring(mx)
+convert(::Type{Dict}, mx::MxArray) = jdict(mx)
+convert(::Type{SparseMatrixCSC}, mx::MxArray) = jsparse(mx)
```

Comment:
> to which function

???

>  it was fine before with `jvalue(mx:MxArray, ::Type)`

I wasn't talking about it but yes it was fine in the sense that they are explicitly too.